root/defaults/nginx/authentik-server.conf

```
@@ -0,0 +1,25 @@
+## Version 2022/09/22 
+# Make sure that your authentik container is in the same user defined bridge network and is named authentik-server
+
+    # all requests to /outpost.goauthentik.io must be accessible without authentication
+    location /outpost.goauthentik.io {
+        proxy_pass              http://authentik-server:9000/outpost.goauthentik.io;
```

```suggestion
        include /config/nginx/resolver.conf;
        set $upstream_authentik authentik-server;
        proxy_pass              http://$upstream_authentik:9000/outpost.goauthentik.io;
```
If you don't use a variable in `proxy_pass` nginx will resolve `authentik-server` on startup and never again. If the `authentik-server` container changes IP addresses later (ex: container recreation) and nginx is not restarted it will not resolve the new container IP.

Using a variable makes nginx check the resolver https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/6e87500cf1768eb628e82a95366c3119a5e53c44/root/etc/s6-overlay/s6-rc.d/init-nginx/run#L51-L52 which will refresh every 30 seconds in our default configuration.